spec/controllers/disbursements_controller_spec.rb

```diff
@@ -27,4 +27,101 @@
       expect(response).to be_successful
     end
   end
+
+  context 'edit' do
+    let(:claim) { instance_double(Claim, id: claim_id) }
+    let(:claim_id) { SecureRandom.uuid }
+    let(:form) { instance_double(DisbursementsForm) }
+    let(:disbursement_id) { SecureRandom.uuid }
+    let(:disbursement) { instance_double(V1::Disbursement, id: disbursement_id, attributes: item) }
+    let(:item) do
+      {
+        'id' => disbursement_id,
+        'miles' => nil,
+        'details' => 'Details',
+        'pricing' => 1.0,
+        'vat_rate' => 0.2,
+        'apply_vat' => 'false',
+        'other_type' => {
+          'en' => 'Apples',
+          'value' => 'Apples'
+        },
+        'vat_amount' => 0.0,
+        'prior_authority' => 'yes',
+        'disbursement_date' => '2022-12-12',
+        'disbursement_type' => {
+          'en' => 'Other',
+          'value' => 'other'
+        },
+        'total_cost_without_vat' => 100.0
+      }
+    end
+
+    before do
+      allow(Claim).to receive(:find).and_return(claim)
+      allow(BaseViewModel).to receive(:build).and_return([disbursement])
+      allow(DisbursementsForm).to receive(:new).and_return(form)
+    end
+
+    context 'when URL is for disburement' do
+      it 'renders successfully with claims' do
+        allow(controller).to receive(:render)
+        get :edit, params: { claim_id: claim_id, id: disbursement_id }
+
+        expect(controller).to have_received(:render)
+                          .with(locals: { claim: claim, form: form, item: disbursement })
+        expect(response).to be_successful
+      end
+    end
+  end
+
+  context 'update' do
+    let(:claim) { instance_double(Claim, id: claim_id) }
+    let(:claim_id) { SecureRandom.uuid }
+    let(:form) { instance_double(DisbursementsForm, save:) }
+    let(:disbursement_id) { '1c0f36fd-fd39-498a-823b-0a3837454563' }
+    let(:disbursement) { instance_double(V1::Disbursement, id: disbursement_id, attributes: item) }
```

Comment:
`attributes: item` this can;t be wrong based on what the next lines does.

It feels like the variable (let) names have gotten confessing - I would recommend a review and work out meaningful names if possible to avoid this.

Reply:
Updated